spec/spec_helper.rb

```diff
@@ -106,4 +106,13 @@
       example.run
     end
   end
+
+  # Reload configuration only from "default" RMT config file to avoid
+  # interference from '/etc/resolv.conf' or 'config/rmt.local.yml' when running
+  # tests randomly.
```

Comment:
:heart: I wanted to do the same in my pr. Can you add a comment about this to https://github.com/SUSE/rmt/blob/disable-drpm-mirror-default/lib/rmt/config.rb#L9 ?

Reply:
Added on 5e4f7be.